### PR TITLE
added babyweights vertex training&deploy lab preprocessing layer version

### PR DESCRIPTION
**Pull request by takumiohym**
Sub task of #194

### CODE REVIEW

**Comment by BenoitDherin** [comment]
When testing locally I am running into the error below because the hypertune module is not installed. Please add it to the requirement.txt so that the following cell can run:

image

**Comment by BenoitDherin** [comment]
When launching the training or tuning job on Vertex AI, the cell seems to queue the job fine but none is triggered on Vertex AI. The reason is I believe the gcloud command is the one targeting CAIP and not Vertex AI. This needs to be fixed so that this lab be run. Here is an example of the command creating the issue:

image

All the commands below this cell have the same behavior or queuing a job but not being able to actually launch it on Vertex AI.

File: notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb [hunk 1]
```diff
@@ -0,0 +1,926 @@
+{
```
Comment (BenoitDherin) [inline notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb:1]: <ul><li>Please add this dependency in the <code>requirements.txt</code> at the repo top-level </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (BenoitDherin) [inline notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb:1]: <p>Same comments as in the previous labs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

File: notebooks/end-to-end-structured/labs/5a_train_keras_vertex_babyweight.ipynb [hunk 2]
```diff
@@ -0,0 +1,743 @@
+{
```
Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/5a_train_keras_vertex_babyweight.ipynb:1]: <p>Add back a mention "Lab Task" or "Exercise" to help locate the exercises and add a brief description of what's asked.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

File: notebooks/end-to-end-structured/labs/5b_deploy_keras_vertex_babyweight.ipynb [hunk 3]
```diff
@@ -0,0 +1,456 @@
+{
```
Comment (BenoitDherin) [inline notebooks/end-to-end-structured/labs/5b_deploy_keras_vertex_babyweight.ipynb:1]: <p>Add back "Lab Task" or "Exercise" to help locate the exercises.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

**Review by BenoitDherin — APPROVED** [review]
LGTM. Great work!

File: notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb (hunk 1 above)
Comment (sanjanalreddy) [reply notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb:1]: <p>+1</p>

Comment (sanjanalreddy) [inline notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #110.        for nc in NUMERICAL_COLUMNS:
</pre><p>nc -&gt; numerical_column</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

**Review by sanjanalreddy — APPROVED** [review]

**Comment by takumiohym** [comment]
Thanks @sanjanalreddy and @BenoitDherin.
I moved cloud-hypertune module to `requirements.txt`, and reflected the other feedbacks. Could you briefly take a look again?

File: notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb (hunk 1 above)
Comment (takumiohym) [reply notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb:1]: <p>I moved it to requirements.txt. Thanks!</p>

Comment (takumiohym) [reply notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb:1]: <p>Done. Thanks.</p>

Comment (takumiohym) [reply notebooks/end-to-end-structured/solutions/5a_train_keras_vertex_babyweight.ipynb:1]: <p>Done. Thanks.</p>

File: notebooks/end-to-end-structured/labs/5b_deploy_keras_vertex_babyweight.ipynb (hunk 3 above)
Comment (takumiohym) [reply notebooks/end-to-end-structured/labs/5b_deploy_keras_vertex_babyweight.ipynb:1]: <p>Done. Thanks.</p>

File: notebooks/end-to-end-structured/labs/5a_train_keras_vertex_babyweight.ipynb (hunk 2 above)
Comment (takumiohym) [reply notebooks/end-to-end-structured/labs/5a_train_keras_vertex_babyweight.ipynb:1]: <p>Done. Thanks.</p>

**Pull request merged**